packages/block-library/src/image/edit.native.js

```diff
@@ -60,6 +60,8 @@ const getUrlForSlug = ( image, { sizeSlug } ) => {
 	return get( image, [ 'media_details', 'sizes', sizeSlug, 'source_url' ] );
 };
 
+const WIDE_ALIGNMENTS = [ 'wide', 'full' ];
```

Comment:
It's created also in `packages/block-editor/src/hooks/align.native.js`, I think it will be helpful to create util for that

Reply:
Will do! Thanks =)